<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportForm</class>
 <widget class="QWidget" name="ExportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="exportSettingsComboBox">
     <item>
      <property name="text">
       <string>Auto (from module size)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Letter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Letter (DL)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Postcard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Business Card</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square 1000 px</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square 2000 px</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DCI 8K 4320p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FUHDTV 8K 4320p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DCI 4K Scope</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DCI 4K Flat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UHDTV 4K 2160p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DCI 2K Scope</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DCI 2K Flat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HDTV 1080p</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HD 720p</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="formFrame">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthEdit">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="heightEdit">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Definition</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="definitionEdit">
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> dpi</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>72</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Transparent Background</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="transparentCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Background Color</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="BGColorEdit">
        <property name="inputMask">
         <string>\#hhhhhh</string>
        </property>
        <property name="text">
         <string>#000000</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="cursorPosition">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Colors</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="colorsComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>Monochrome (1 bit)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greyscale (8bpc)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Colors (8bpc)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
